rosidl_cmake/cmake/rosidl_export_typesupport_targets.cmake

```diff
@@ -44,7 +46,10 @@ macro(rosidl_export_typesupport_targets)
           "not-imported targets")
       endif()
 
-      list(APPEND _ROSIDL_CMAKE_EXPORT_TYPESUPPORT_TARGETS "${PROJECT_NAME}::${_target}")
+      list(APPEND
+        _ROSIDL_CMAKE_EXPORT_TYPESUPPORT_TARGETS
+        "${library_suffix}:${_target}"
```

Comment:
This looks wrong to me. Modern CMake targets have (by convention) the form `<pkgname>::<libname>`. The library suffix seems to be the `_target_suffix` from the caller which has two underscores and the typesupport package name? Where is this target being defined?

Reply:
> This looks wrong to me. Modern CMake targets have (by convention) the form `<pkgname>::<libname>`

That hasn't changed.
Please read relevant changes in other files:

https://github.com/ros2/rosidl/blob/d42b3925208871cabd0385282112f73cf14f52a0/rosidl_cmake/cmake/rosidl_cmake_export_typesupport_targets-extras.cmake.in#L11-L15

Reply:
From the change it simply wasn't clear to me that you don't pass targets anymore but a tuple.

Reply:
> From the change it simply wasn't clear to me that you don't pass targets anymore but a tuple.

yes, I understand. Should I change the variable `_ROSIDL_CMAKE_EXPORT_TYPESUPPORT_TARGETS` name to something different?

Reply:
You could rename the variable - I just don't have a good naming proposal. It can stay as is too.